.github/workflows/dependabot.yml

```
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you’ll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
```

actionlint (1.7.4)

unexpected key "version" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)